Wrapping/Generators/Doc/CMakeLists.txt

```diff
@@ -109,4 +109,8 @@ macro(itk_end_wrap_submodule_DOC module_name)
   #    COMMENT "-- Wrapping library ${module_name}: Generating swig interface for inline documentation."
     )
     list(APPEND ITK_WRAP_DOC_DOCSTRING_FILES ${swig_doc_interface_file})
+
+    unset(swig_doc_interface_file)
```

Comment:
A few of these `unset` commits could be squashed together.

Reply:
Done.